my-app/src/app/components/header/header.component.spec.ts

```diff
@@ -20,4 +30,12 @@ describe('HeaderComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should call console.log and logout method of AuthService by onLogOff()', () => {
```

Comment:
The description still mentions `console`

Reply:
Fixed in next pr (https://github.com/Russzh/angular_course/pull/6)